package.json

```diff
@@ -12,21 +12,21 @@
   "repository": "https://github.com/single-spa/vue-cli-plugin-single-spa",
   "bugs": "https://github.com/single-spa/vue-cli-plugin-single-spa/issues",
   "dependencies": {
-    "semver": "^7.3.2",
-    "standalone-single-spa-webpack-plugin": "^1.2.0",
-    "systemjs-webpack-interop": "^2.3.1"
+    "semver": "^7.3.5",
+    "standalone-single-spa-webpack-plugin": "^2.0.2",
+    "systemjs-webpack-interop": "^2.3.7"
   },
   "husky": {
     "hooks": {
       "pre-commit": "pretty-quick --staged && eslint ."
     }
   },
   "devDependencies": {
-    "eslint": "^7.7.0",
+    "eslint": "^7.28.0",
     "eslint-config-important-stuff": "^1.1.0",
-    "husky": "^4.2.5",
-    "prettier": "^2.0.5",
-    "pretty-quick": "^2.0.1"
+    "husky": "^6.0.0",
```

Comment:
This upgrade will break the git hooks - the 4->5 upgrade requires creating a new folder and files rather than using the `"husky"` property in the package.json. Additionally, it sometimes requires adding `pinst` to avoid installing git hooks for users of vue-cli-plugin-single-spa.

If you're up to making this upgrade, feel free to. Or you can stay at husky@4 if you'd rather not.